pytest.ini

```diff
@@ -1,3 +1,3 @@
 [pytest]
-testpaths = listenbrainz_spark
+testpaths = tests listenbrainz_spark/tests listenbrainz_spark/recommendations
```

Comment:
All code should be inside the `listenbrainz_spark` module because it makes imports etc easier. I think you should just move the file you have in `tests` to `listenbrainz_spark/tests`.

For context, we've been burned by not having code in a single module leading to us having to add stuff to `sys.path` and needing major refactors.

Reply:
If you were doing this because of import problems while putting tests in directories other that `listenbrainz_spark/test`, I've fixed that problem in #72. Please take a look.